csv2bufr/__init__.py

```diff
@@ -651,12 +651,13 @@ def transform(data: str, metadata: dict, mappings: dict,
         result["_meta"] = {
             "identifier": rmk,
             "md5": rmk,
+            "wigos_id": metadata['wigosIds'][0]['wid'],
             "data_date": message.get_datetime(),
             "originating_centre": message.get_element("bufrHeaderCentre"),
             "data_category": message.get_element("dataCategory")
         }
-        time = datetime.now(timezone.utc).isoformat()
-        LOGGER.info(f"|{time}|{metadata['wigosIds'][0]['wid']}|{result['_meta']['data_date']}|{rmk}")  # noqa
+        time_ = datetime.now(timezone.utc).isoformat()
+        LOGGER.info(f"{time_}|{result['_meta']}")
```

Comment:
Can we include the WIGOS ID in the log output, this may come in useful in the future.

Reply:
Good point.  Given `wigos_id` is now part of `_meta`, it will also appear as part of the log output.